src/helpers/formatCreationDate.js

```diff
@@ -0,0 +1,4 @@
+export const formatCreationDate = (creationDate) => {
```

Comment:
great!

(a more simple one:)
```suggestion
export const formatCreationDate = (creationDate) => creationDate.replaceAll(/\//g, '.');
```

are you familiar with this syntax? if it is only one line, you can immediately return it